collect_app/src/main/java/org/odk/collect/android/preferences/ServerPreferencesFragment.java

```diff
@@ -99,10 +102,20 @@ public static ServerPreferencesFragment newInstance(boolean adminMode) {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         addPreferencesFromResource(R.xml.server_preferences);
-
         initProtocolPrefs();
     }
 
+    @Override
+    public void onDestroyView() {
+        serverUrlPreference = null;
```

Comment:
Not sure why you need this. If you're getting a leak here lets try to work out what's causing and see if there is a better way to solve it!